test/net/sourceforge/plantuml/tim/stdlib/GetenvTest.java

```diff
@@ -0,0 +1,59 @@
+package net.sourceforge.plantuml.tim.stdlib;
+
+import net.sourceforge.plantuml.tim.EaterException;
+import net.sourceforge.plantuml.tim.EaterExceptionLocated;
+import net.sourceforge.plantuml.tim.expression.TValue;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.Collections;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Tests the internal function %getenv.
+ */
+class GetenvTest {
+
+	/**
+	 * Tests getenv should not publish plantuml.security.* environment variables.
+	 *
+	 * @throws EaterException should not
+	 * @throws EaterExceptionLocated should not
+	 */
+	@ParameterizedTest
+	@ValueSource(strings = {
+			"plantuml.security.blabla",
+			"plantuml.SECURITY.blabla",
+			"plantuml.security.credentials.path",
+	})
+	void executeReturnFunctionSecurityTest(String name) throws EaterException, EaterExceptionLocated {
+		System.setProperty("plantuml.security.blabla", "example");
+		Getenv cut = new Getenv();
```

Comment:
Hi @arittner, 

Just a _minor_ naming question:
- Why is this variable named `cut`?

Regards,
Th.

Reply:
_I discovered the java world... _
I must to learn all.... from the beginning....
:-)
Thanks